MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,60 @@
+# v0.x.x &rightarrow; v1.0.0-beta.0
+
+Full documentation for v1 endpoints is available here: https://paritytech.github.io/substrate-api-sidecar/dist/
+
+For most users, the major bump will require transitioning all the endpoint paths
+they use to an updated version. The v1.0.0.beta.0 release contains both the old
+and new endpoints to help facilitate the transition.
+
+**`substrate-api-sidecar` releases after Friday, October 2nd, 2020, will not
```

Comment:
This might be a little aggressive on time. Maybe do one more week.

Reply:
See updated. I moved it back to October 16th.